Framework/CurveFitting/src/Functions/AsymmetricPearsonVII.cpp

```diff
@@ -0,0 +1,214 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2022 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidCurveFitting/Functions/AsymmetricPearsonVII.h"
+#include "MantidAPI/FunctionFactory.h"
+#include "MantidCurveFitting/Constraints/BoundaryConstraint.h"
+
+#include <cmath>
+#include <iostream>
+
+namespace Mantid::CurveFitting::Functions {
+
+using namespace CurveFitting;
+using namespace Constraints;
+using namespace std;
+using namespace Kernel;
+using namespace API;
+
+DECLARE_FUNCTION(AsymmetricPearsonVII)
+
+AsymmetricPearsonVII::AsymmetricPearsonVII() : IPeakFunction() {}
+
+void AsymmetricPearsonVII::init() {
+  declareParameter("PeakHeight", 0.0, "Hight of the peak");
+  declareParameter("PeakCentre", 0.0, "Location of the peak");
+  declareParameter("Width", 0.0, "Full width at half maximum");
```

Comment:
can we have some typical values for these 3? A default of zero for height is not helpful as it will struggle to find anything without being updated.

Reply:
sure, I've modified that